catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/TransformersCommandTest.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.commands.catalog;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.osgi.framework.BundleContext;
+import org.osgi.framework.ServiceReference;
+
+import ddf.catalog.transform.InputTransformer;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TransformersCommandTest extends ConsoleOutputCommon {
+
+    private static final String SERVICE_ID = "id";
+
+    private static final String MIME_TYPE = "mime-type";
+
+    private static final String SCHEMA = "schema";
+
+    @Mock
+    BundleContext bundleContext;
+
+    TransformersCommand transformersCommand;
+
+    @Before
+    public void setUp() {
+        transformersCommand = new TransformersCommand();
+        transformersCommand.setBundleContext(bundleContext);
+    }
+
+    @Test
+    public void testNullIds() throws Exception {
+
+        List<ServiceReference<InputTransformer>> serviceReferences = new ArrayList<>();
+        ServiceReference<InputTransformer> serviceReference = mock(ServiceReference.class);
+        serviceReferences.add(serviceReference);
+
+        when(serviceReference.getProperty(SERVICE_ID)).thenReturn(null);
+        when(bundleContext.getServiceReferences(InputTransformer.class, null)).thenReturn(
+                serviceReferences);
+
+        transformersCommand.executeWithSubject();
+        assertThat(consoleOutput.getOutput(), containsString("There are no active transformers"));
```

Comment:
✏️ It might be a bit less brittle for testing purposes if you convert the string messages you'll be testing for here into package-private consts in the `TransformersCommand` class. This way, if the wording of the messages ever changes, your unit tests will remain in sync.

We don't traditionally do that, but commands are also an outlier in that they're the odd birds in which we test output like this.